scoring/templates/scoring/methodology.html

```diff
@@ -546,21 +540,26 @@ <h4 class="mb-3">County Council Joint Plans</h4>
                         For the questions listed below, the evidence had to be included in the District plan to get the mark.</p>
                     <ul class="mb-3">
                         {% comment %} TODO Include acnhors to the questions. How can we do this dynamic? {% endcomment %}
```

Comment:
`{% url 'question' 's1_gov_q2' %}` should do it, for example. Just note:

1. The second argument (the question code, `'s1_gov_q2` in the example above) should be lowercase, not uppercase.
2. You’ll have to link to the main question page (eg `s1_gov_q2` in the example above), because we don’t have a separate page for the `_sp` sub-questions.